src/App.js

```diff
@@ -4,15 +4,57 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>
-          Eric F. Olsen
-        </h1>
-        <h3>
-          Software Engineer
-        </h3>
-      </header>
+      <Header />
+      {/* <Story />
+      <Resume />
+      <Projects />
+      <Testimonials /> */}
+    </div>
+  );
+}
+
+function Header() {
+  return (
+    <header className="App-header">
+      <img src={logo} className="App-logo" alt="logo" />
+      <h1>
+        Eric F. Olsen
+      </h1>
```

Comment:
You could put <h1>...</n1> and the other headers on 1 line

Reply:
I went ahead and put <h1>Eric F. Olsen</h1> on one line, and <h3>Software Engineer</h3> on a separate line, I feel that is more readable. Is there a reason to put them both on the same line?

Reply:
Based on your other comment, I've scrapped the h3 and replaced it with p and a css class to adjust it.